### PR TITLE
Adding raspbian support

### PR DESCRIPTION
**Pull request by ampledata**
Adding support raspbian (Debian on Raspberry PI).

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -71,11 +71,11 @@
 
 init_template_dir = value_for_platform_family(
   ["rhel", "fedora"] => "rhel",
-  "debian" => "debian"
+  ["debian", "raspbian"] => "debian"
```
Comment (coderanger) [inline recipes/default.rb]: Does it really show as its own platform_family?

Comment (coderanger) [inline recipes/default.rb]: No, family is still 'debian' https://github.com/chef/ohai/blob/master/lib/ohai/plugins/linux/platform.rb#L114-L115

**Comment by pelletiermaxime** [comment]
I was about to do the exact same PR (minus the platform_family :P), so this would be great for me.

**Comment by ampledata** [comment]
Feel free to resubmit with correct code.

**Pull request closed without merging**